README.md

```diff
@@ -1,62 +1,28 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
+# Hi Microverse!!
 
 > Description the project.
```

Comment:
As part of documenting your project professionally, the README.md file is the key to this. The following areas seem to not have been addressed as you have deleted them from the file.
- [ ] The top Microverse tag that is required in all Microverse projects is to be present. Kindly re-add this at the top  ```![](https://img.shields.io/badge/Microverse-blueviolet)```
- [ ] The "Built with" section is empty. Kindly list what you used to build this project. eg HTML, CSSS, linters etc
- [ ] Kindly customize your MIT.md file to include your details that are relevant for this specific project